src/assets/GraphqlAsset.js

```diff
@@ -0,0 +1,22 @@
+const Asset = require('../Asset');
+const localRequire = require('../utils/localRequire');
+
+class GraphqlAsset extends Asset {
+  constructor(name, pkg, options) {
+    super(name, pkg, options);
+    this.type = 'js';
+  }
+
+  async parse(code) {
+    let gql = await localRequire('graphql-tag', this.name);
```

Comment:
Question: does this work if `graphql` and `graphql-tag` are not already installed? In that case `graphql-tag` would be autoinstalled by parcel, but not `graphql` which is a peer dependency. Would that cause it to break?

Perhaps we will need to extend the autoinstall mechanism to also install peer dependencies... Or I guess we could just `localRequire('graphql')` here as well, which would force an install.

Reply:
@devongovett Ah, good catch. No, it won't work if they aren't already installed. Fortunately, the error message will be along the lines of "cannot find `graphql`". My preference is to enhance auto install to check and handle peer dependencies. 

Adding `localRequire('graphql')` would be an unclear side-effect in the code. If `graphql-tag` ever drops it as a peer dependency, we'd keep installing it for no reason.

Reply:
Ok, let's do that as a separate PR.